galsim/chromatic.py

```diff
@@ -1143,11 +1234,14 @@ def drawImage(self, bandpass, image=None, integrator='trapezoidal', **kwargs):
 
         @returns the drawn Image.
         """
+        # Store the last bandpass used.
+        self._last_bp = bandpass
         if self.SED.dimensionless:
             raise ValueError("Can only draw ChromaticObjects with spectral SEDs.")
 
         int_im = self._get_interp_image(bandpass, image=image, integrator=integrator, **kwargs)
         image = int_im.drawImage(image=image, **kwargs)
+        self._last_wcs = image.wcs
```

Comment:
These two lines aren't covered, which means this whole function is never being called in the tests.  Not directly related to this PR I guess, but probably worth doing.